nntrainer/include/layer_internal.h

```diff
@@ -83,7 +83,10 @@ class Layer : public ml::train::Layer {
     trainable(trainable_),
     num_weights(0),
     num_inputs(1),
-    num_outputs(1) {}
+    num_outputs(1) {
```

Comment:
I think we should get rid of those two, as vector has it's own size, this violates Single source of truth principle

Reply:
I'll make an issue to deal separately